### DISCUSSION TITLE
[Idea]: Scene Switching Issue with M3 and OBS Studio During Zoom Screen Sharing

### DISCUSSION THREAD

**Discussion by bucmado** [sircharlo/meeting-media-manager#3813] [Q&A] [answered]
### Is your idea related to a problem?

I'm facing an issue when using M3 alongside OBS Studio to share images and videos during Zoom meetings. The problem occurs when I switch from one image to another in M3 without using the pause option. In this case, OBS Studio automatically switches scenes, causing Zoom participants to stop seeing the shared content, which interrupts the presentation.

Currently, the only temporary solution I've found is to stop or pause screen sharing in Zoom and then restart it.

It would be ideal if the M3 system allowed transitions between images or videos within the same playback session without triggering an automatic scene change in OBS Studio. This would improve the experience, ensuring continuity of the presentation and avoiding interruptions.

I would appreciate any ideas or suggestions from anyone who has encountered this issue or knows a way to resolve it.

Thanks in advance for your help!

### Describe your idea

N/A

### Describe alternatives you've considered

N/A

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing issues to make sure my idea is not a duplicate.
- [X] I confirm that I have searched through the existing discussions to make sure my idea is not a duplicate.

**Comment by bucmado** [comment]
https://github.com/user-attachments/assets/c16a41d4-cf02-4291-b515-0a71eb88c20d

**Reply by e-Feitosa** [reply, marked as answer]
Your 'OBS Studio' cannot be in STUDIO MODE, the m3 works as shown in the image below:

![image](https://github.com/user-attachments/assets/48c6b074-d528-4cca-8033-b4d992e231b6)

**Comment by bucmado** [comment]
Thank you so much for your prompt and precise response! Disabling STUDIO MODE in OBS Studio resolved the issue perfectly, and now everything works as expected with M3. I truly appreciate your clear guidance and the included image, which made it easy to understand and implement the solution.

Thanks again for your support! 😊
